draft-ietf-rats-msg-wrap.md

```diff
@@ -652,9 +652,8 @@ The following subsections provide further elaboration on these points, particula
 
 CMW itself does not provide any mechanisms for authenticity, integrity protection, or confidentiality.
```

Comment:
Wouldn't the signed-cmw satisfy the authenticity and integrity protection properties?

Reply:
Added in [d26fe34](https://github.com/ietf-rats-wg/draft-ietf-rats-msg-wrap/pull/223/commits/d26fe34b6ad1f7c355dd02280e8bd79b8ca6d21c)